test/control_filters/test_low_pass_filter.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test_low_pass_filter.hpp"
+
+TEST_F(LowPassFilterTest, TestLowPassFilter)
```

Comment:
Excellent test. Please also add a `load_low_pass_filter_test.cpp` as we have for the controllers. This tests if the plugin is loadable. The best is to add a new file for this.

For an example see [here](https://github.com/ros-controls/ros2_controllers/blob/master/forward_command_controller/test/test_load_forward_command_controller.cpp)